test/integration/features/basic.feature

```diff
@@ -0,0 +1,20 @@
+# 
+Feature: Bringup
+  In order to use MiniShift
+  As a user
+  I need to be able to bring up a test environment
+
+  Scenario: Docker environment variables
+    Given a "developer"
+     When executing "minishift start --docker-env=FOO=BAR --docker-env=BAZ=BAT"
+      And should be running
+     Then should have a valid IP address
+     Then executing "minishift ssh echo hello" should contain
+      """
+      hello
+      """
+     Then executing "minishift ssh cat /var/lib/boot2docker/profile" should contain
```

Comment:
This really mixes condition and testable outcome. I'd like so see more something like this:

    Feature: Bringup
      In order to use MiniShift
      As a user
      I need to be able to bring up a test environment
    
      Scenario: Docker environment variables
        Given a "developer"
         When executing "minishift start --docker-env=FOO=BAR --docker-env=BAZ=BAT"
         Then Minishift should be running
         And  it should have a valid IP address
         
         When executing "minishift ssh echo hello"
         Then stdout from "minishift ssh echo hello" should contain
          """
          hello
          """
    
         When executing "minishift ssh cat /var/lib/boot2docker/profile" 
         Then stdout from "minishift ssh cat /var/lib/boot2docker/profile" should contain
          """
          export "FOO=BAR"
          export "BAZ=BAT"
          """    

In this case, the executing step would execute the command and store exit status, stdout and stderr output in a map or so. Then I can have as many assertions as I want after the execution. I can not only assert that something specific comes out stdout, but also that stderr was empty and exist status was '0', eg:

         When executing "minishift ssh cat /var/lib/boot2docker/profile" 
         Then stdout from "minishift ssh cat /var/lib/boot2docker/profile" should contain
          """
          export "FOO=BAR"
          export "BAZ=BAT"
          """    
          And stderr should be empty
          And exit status should be 0

In vagrant-service-manager this abstraction was provided by Aruba, but the basic capturing of output and exit code, should be easy enough to implement.